lib/parsers/txt.mjs

```diff
@@ -132,6 +161,8 @@ export async function parse (rawText, filename) {
       }
     }
 
+    data.boilerplate.similar2119boilerplate = hasBoilerplateMatch(normalizedText, BOILERPLATE_PARTS.rfc2119, BOILERPLATE_PARTS.rfc2119_alt, BOILERPLATE_PARTS.rfc8174)
```

Comment:
This seems strange - why should `similar2119boilerplate` be true if the document actually contains 2119 or 8174 boilerplate? What breaks if this only tests for a `rfc2119_alt` match?

Reply:
Later in the validation function, I do check whether the document contains boilerplate. I agree that it might not be ideal to immediately set similar2119boilerplate to true if we’ve already found a full-fledged boilerplate (e.g., rfc2119 or rfc8174).
I currently check all three variants (rfc2119, rfc2119_alt, rfc8174) because they differ slightly, and I need to support all these cases. However, if there’s a specific requirement to only validate against one of them (for instance, rfc2119_alt), I can adjust the check accordingly. This would simplify the logic and make the result more focused on a specific goal.
Should I limit the check to just one variant, or is it better to keep supporting all three?

Reply:
you'll need all three for other checks, but for the check that says 'you have 2119 keywords, but no 2119 or 8174 boilerplate, but something similar to 2119 was found' it only makes sense to match 2119_alt (or other possible future variants). It doesn't makes sense to say "you have no 2119" if you've _actually found_ and exact 2119 match.